drivers/gpu/drm/vc4/vc4_plane.c

```diff
@@ -1248,6 +1248,10 @@ static int vc4_plane_mode_set(struct drm_plane *plane,
 	case DRM_FORMAT_MOD_BROADCOM_SAND256: {
 		uint32_t param = fourcc_mod_broadcom_param(fb->modifier);
 
+		if (param == SAND_AUTO_HEIGHT) {
```

Comment:
The braces don't add anything.

Reply:
Indeed.

I did have it as an `if {   } else if (param > SCALER_TILE_HEIGHT_MASK)`, hence matching the second clause having braces, but then viewed the check to be valid on the computed value too.

Fixed.